src/SimplCommerce.WebHost/wwwroot/themes/CozaStore/js/cozastore.js

```diff
@@ -210,4 +210,21 @@
         $('.js-modal1').removeClass('show-modal1');
     });
 
+    $('a.lang-selector').on('click', function (e) {
```

Comment:
Why we need this?

Reply:
_SelectLanguagePartial of the "Coza" theme is different from the default and the function in main.js does not work for this theme.
![Coza localization](https://user-images.githubusercontent.com/867631/58650919-50ba8200-82e6-11e9-85dc-5a64e17ea60d.png)